templates/donate.html

```diff
@@ -66,144 +67,8 @@
-                                        <svg class="card-social-icon">
-                                            <title>{{ post.extra.steam_developer }} on Steam</title>
-                                            <use href="#steam" />
-                                        </svg>
-                                    </a>
-                                </div>
-                            {% endif %}
-                            {% if post.extra.website %}
-                                <div>
-                                    <a href="{{ post.extra.website }}">
-                                        <svg class="card-social-icon">
-                                            <title>{{ post.extra.website }}</title>
-                                            <use href="#globe" />
-                                        </svg>
-                                    </a>
-                                </div>
-                            {% endif %}
-                        </div>
-                        {% if post.extra.roles %}
-                        <div class="people-roles">
-                            {% for role in post.extra.roles %}
-                                {% if role == "Project Lead" %}
-                                    <div class="people-role people-role-card people-role-project-lead">{{ role }}</div>
-                                {% elif role == "Maintainer" %}
-                                    <div class="people-role people-role-card people-role-maintainer">{{ role }}</div>
-                                {% elif role is starting_with("SME-") %}
-                                    <div class="people-role people-role-card people-role-sme">{{ role }}</div>
-                                {% else %}
-                                    <div class="people-role">{{ role }}</div>
-                                {% endif %}
-                            {% endfor %}
-                        </div>
-                        {% endif %}
-                        {% if post.extra.bio %}
-                            <div class="card-text-dense">
-                                {{ post.extra.bio }}
-                            </div>
-                        {% endif %}
-                        {% if post.extra.sponsor %}
-                            <div style="position: absolute; bottom: 0.4rem; right: 0.8rem;">
-                                <svg class="card-social-icon card-sponsor-icon">
-                                    <use href="#heart" />
-                                </svg>
-                                <a href="{{ post.extra.sponsor }}">Sponsor me!</a>
-                            </div>
-                        {% endif %}
-
-                    </div>
-                </div>
-            </div>
-
-            {% endfor %}
-        </div>
-        {% endif %}
+        {% set members = load_data(path="org_members.json") %}
 
+        {{ members_macro::members_grid(members=members) }}
```

Comment:
```suggestion
        {{ members_macro::members_grid(members=members | filter(attribute="sponsor")) }}
```
Only members with a sponsor link should be displayed here